### ISSUE TITLE
Error when opening on Windows 8.1 / 'unable to start correctly (0xc000007b)'

### ISSUE THREAD

**Issue by szszszsz** [dfu-programmer/dfu-programmer#50]
Hi!

I am trying now to update the firmware under the Windows 8.1 and I keep the messages about `dfu-programmer` being broken. I have copied the libusub0 dlls to the app's directory (from all architectures), but it has not helped. Error is shown both with and w/o Admin privileges.

OS Windows 8.1 x64

Error looks like this:
```
---------------------------
dfu-programmer.exe - Application Error
---------------------------
The application was unable to start correctly (0xc000007b). Click OK to
close the application.
---------------------------
OK
---------------------------
```

And without the library copied:
```
The program can't start because libusb0.dll is missing from your computer. 
Try reinstalling the program to fix this problem. 
```
Just for completeness - I have downloaded it from project's page [2] -> [1]

[1]
https://sourceforge.net/projects/dfu-programmer/files/dfu-programmer/0.7.0/
[2] https://dfu-programmer.github.io/

PS Same with 0.7.2

**Comment by szszszsz** [comment]
Sorry, I have created empty issue by mistake (pushing enter does it on title form input). Now the description is updated.

cc @jans23

**Comment by Zeigren** [comment]
You shouldn't need the libusb dlls in the same directory as dfu-programmer, you should only need to have the libusb driver installed.

You should be able to update the driver by plugging in your device, hitting reset or whatever on it to have it jump to the bootloader, then go to device manager and it should show up as the name of the micro-controller, update the driver and browse to the dfu-prog-usb-1.2.2 folder.

Hopefully that works and it isn't some other weird problem.

**Comment by szszszsz** [comment]
Thank you for the tip! I will let you know once I will check it.

**Comment by KonstantineF** [comment]
Just wanted to update this thread to say that @Zeigren you are correct. I had the exact same issues with @szszszsz but I managed to use dfu-programmer by following your solution. The steps were there in the process I followed but I naively tried to use the programmer on its own as some documents mentioned "installing" so I tried to run the .exe file without having it connected to a hardware device (in my case ATMEGA). 

Plugging the device and had to go to Device Manager, and update the drivers but everything run smoothly.
